### PR TITLE
Backport material properties from ResStock-HPXML

### PR DESCRIPTION
**Pull request by aspeake**
## Pull Request Description

Update exterior finish absorptance and wall density values to align with ResStock-HPXML values. Also removes sheathing layer from CMU and brick wall types.

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [ ] All rake tasks have been run, and pass
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] No unexpected regression test changes
- [ ] All tests are passing
- [ ] The [changelog](https://github.com/NREL/resstock/blob/develop/CHANGELOG.md) has been updated appropriately
- [ ] This branch is up-to-date with develop

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: resources/measures/HPXMLtoOpenStudio/resources/materials.rb [hunk 1]
```diff
@@ -279,11 +301,11 @@ def self.RoofingAsphaltShinglesMed
   end
 
   def self.RoofingAsphaltShinglesLight
-    return self.RoofMaterial('Asphalt Shingles, Light', 0.91, 0.8)
+    return self.RoofMaterial('Asphalt Shingles, Light', 0.91, 075)
```
Comment (shorowit) [inline resources/measures/HPXMLtoOpenStudio/resources/materials.rb:304]: @aspeake Typo.

File: resources/measures/HPXMLtoOpenStudio/resources/materials.rb [hunk 2]
```diff
@@ -185,52 +185,74 @@ def self.Concrete(thick_in)
...
   def self.ExtFinishStuccoMedDark
-    return new(name = 'Stucco, Medium/Dark', thick_in = 1.0, mat_base = BaseMaterial.Stucco, k_in = nil, rho = nil, cp = nil, tAbs = 0.9, sAbs = 0.75, vAbs = 0.75)
+    sAbs = self.solar_abs_map['MediumDark']
+    return new(name = 'Stucco, Medium/Dark', thick_in = 1.0, mat_base = BaseMaterial.Stucco, k_in = nil, rho = nil, cp = nil, tAbs = 0.9, sAbs = sAbs, vAbs = sAbs)
   end
 
   def self.ExtFinishStuccoLight
-    return new(name = 'Stucco, Light', thick_in = 1.0, mat_base = BaseMaterial.Stucco, k_in = nil, rho = nil, cp = nil, tAbs = 0.45, sAbs = 0.75, vAbs = 0.75)
+    sAbs = self.solar_abs_map['Light']
+    return new(name = 'Stucco, Light', thick_in = 1.0, mat_base = BaseMaterial.Stucco, k_in = nil, rho = nil, cp = nil, tAbs = 0.45, sAbs = sAbs, vAbs = sAbs)
```
Comment (joseph-robertson) [inline resources/measures/HPXMLtoOpenStudio/resources/materials.rb:205]: `tAbs = 0.45` seems wrong.

Comment (joseph-robertson) [reply resources/measures/HPXMLtoOpenStudio/resources/materials.rb:205]: This [comment](https://github.com/NREL/resstock/pull/561#issuecomment-861804196) implies solar absorptance should be 0.45.

File: resources/measures/HPXMLtoOpenStudio/resources/materials.rb [hunk 3]
```diff
@@ -279,11 +301,11 @@ def self.RoofingAsphaltShinglesMed
   end
 
   def self.RoofingAsphaltShinglesLight
-    return self.RoofMaterial('Asphalt Shingles, Light', 0.91, 0.8)
+    return self.RoofMaterial('Asphalt Shingles, Light', 0.91, 0.75)
```
Comment (joseph-robertson) [inline resources/measures/HPXMLtoOpenStudio/resources/materials.rb:304]: Can we use the same map here?

Comment (aspeake) [reply resources/measures/HPXMLtoOpenStudio/resources/materials.rb:304]: Absorptance for roofing materials are dependent on both material (shingle) and color (light) in restructure-v3, so the exterior wall map isn't applicable and hard-coding was just as efficient. https://github.com/NREL/resstock/blob/restructure-v3/resources/hpxml-measures/HPXMLtoOpenStudio/resources/constructions.rb#L1135

File: resources/options_lookup.tsv [hunk 4]
```diff
@@ -626,20 +626,20 @@ Insulation Wall	"Double Wood Stud, R-45, Grade 3"	ResidentialConstructionsWallsD
...
-Insulation Wall	"Brick, 12-in, 3-wythe, R-19"	ResidentialConstructionsWallsCMU	thick_in=12	conductivity=5.29	density=49	framing_factor=0.076	furring_r=19	furring_cavity_depth_in=1	furring_spacing=24	drywall_thick_in=0.5	rigid_r=0.0																																				
-Insulation Wall	"CMU, 12-in Hollow"	ResidentialConstructionsWallsCMU	thick_in=12	conductivity=5.29	density=49	framing_factor=0.076	furring_r=0	furring_cavity_depth_in=1	furring_spacing=24	drywall_thick_in=0.5	rigid_r=0.0																																				
-Insulation Wall	"CMU, 12-in Hollow, R-10"	ResidentialConstructionsWallsCMU	thick_in=12	conductivity=5.29	density=49	framing_factor=0.076	furring_r=10	furring_cavity_depth_in=2	furring_spacing=24	drywall_thick_in=0.5	rigid_r=0.0																																				
+Insulation Wall	"CMU, 6-in Concrete Filled"	ResidentialConstructionsWallsCMU	thick_in=6	conductivity=5.33	density=140	framing_factor=0.076	furring_r=0	furring_cavity_depth_in=1	furring_spacing=24	drywall_thick_in=0.5	rigid_r=0.0
```
Comment (aspeake) [inline resources/options_lookup.tsv:629]: Using values from OS-HPXML for [brick](https://github.com/NREL/OpenStudio-HPXML/blob/master/HPXMLtoOpenStudio/resources/materials.rb#L407) and [concrete](https://github.com/NREL/OpenStudio-HPXML/blob/master/HPXMLtoOpenStudio/resources/materials.rb#L358) for density inputs.

File: resources/measures/HPXMLtoOpenStudio/resources/materials.rb [hunk 5]
```diff
@@ -185,52 +185,74 @@ def self.Concrete(thick_in)
     return new(name = "Concrete #{thick_in} in.", thick_in = thick_in, mat_base = BaseMaterial.Concrete, k_in = nil, rho = nil, cp = nil, tAbs = 0.9)
   end
 
+  def self.solar_abs_map
```
Comment (aspeake) [inline resources/measures/HPXMLtoOpenStudio/resources/materials.rb:188]: Pulled data from here: https://github.com/NREL/OpenStudio-HPXML/blob/a2aac431ef2776d1a7bfe2920dc350792f781c01/HPXMLtoOpenStudio/resources/constructions.rb#L1193

**Review by joseph-robertson — APPROVED** [review]

**Pull request merged**